db/seeds.rb

```diff
@@ -46,11 +46,14 @@ def create_borrowers(quantity)
   end
 
   def create_categories(quantity)
-    quantity.times do
-      title = Faker::Lorem.words(2).join(" ")
-      description = Faker::Lorem.sentence
-      category = Category.create(title: title, description: description)
-      puts "created category #{category.title}"
+    #quantity.times do
```

Comment:
Incorrect indentation detected (column 4 instead of 6).<br>Missing space after #.